app/layout.tsx

```diff
@@ -10,11 +10,5 @@ export default function RootLayout({
 }: {
   children: React.ReactNode;
 }) {
-  return (
-    <html lang="en">
-      <body className="grid justify-center h-screen items-center bg-teal mx-8">
-        {children}
-      </body>
-    </html>
-  );
+  return <html lang="en">{children}</html>;
```

Comment:
With this change, You'll need to remove the html tag from the (auth)/layout and the (protected)/layout as well, otherwise you'll have nested html tags in you your layout. 

Also just a small remark it might be easier to keep track if not all the layouts are called "RootLayout" :)

Reply:
Okay, fixed it!